scss/_patterns_subnav.scss

```diff
@@ -50,11 +50,17 @@
       @extend %subnav-item--dark-theme;
     }
 
-    .p-subnav.is-light {
+    .p-subnav.is-light,
+    .p-navigation.is-light .p-subnav {
       &::after {
         @include vf-icon-chevron;
       }
 
+      .p-subnav__items,
+      .p-subnav__items--right {
+        @extend %vf-is-bordered;
```

Comment:
Not sure %vf-is-bordered is a good idea. The goal of the theming is to set only colours, keeping things like border-width in the rules defined above. This way, it wouldn't need to override the colour set in this placeholder.

Reply:
besides, the colours of these lines for both themese should already be set by `.p-navigation.is-dark .p-navigation__link>a::before` defined in p-navigation. So not sure why the borders are set here at all?

Reply:
@lyubomir-popov 
I think the current implementations is so confusing because we have border around subnav in light theme, but don't have the border in the dark theme. This causes issues when `is-dark` is applied and border is not removed.

I'll try to change it, so that the border is used in dark theme, but has same colour as background. To make it consistent and easier to theme.